index.html

```diff
@@ -28,32 +31,55 @@
                     <div class="code highlight">
                         <span class="kr">read</span>
                         <span id="log-example-adapter" class="adapter" style="width: 60px;"></span>
-                        -last <span class="s s-Moment">:1 day:</span> status_code = <span class="mi">500</span>
+                        -last <span class="s s-Moment">:day:</span> status_code=<span class="mi">500</span><br>
                         <span class="o">|</span> <span class="kr">reduce</span> -every
-                        <span class="s s-Moment">:1m:</span> count()
+                        <span class="s s-Moment">:1m:</span> count()<br>
+                        <span class="o">|</span> <span class="kr">view</span> table
                     </div>
                     <p class="description">
                         Find all the log lines with status_code 500 in the last
-                        day and return the per-minute counts - either from
+                        day and return the per-hour counts - either from
                         splunk, a file, or elastic.
                     </p>
                 </div>
                 <div class="juttle-example">
                     <h3>
-                        Moving averages are just one line away:
+                        Mixed historical / live moving averages are just one line away:
                     </h3>
                     <div class="code highlight">
                         <span class="kr">read</span>
                         <span id="moving-average-example-adapter" class="adapter" style="width: 80px;"></span>
-                        -last <span class="s s-Momemnt">:6 hours:</span>
+                        -from <span class="s s-Moment">:1 hour ago:</span> -to <span class="s s-Moment">:end:</span>
                         name = <span class="s2">'cpu'</span>
                         <br />
                         <span class="o">|</span> <span class="kr">reduce</span> -over
                         <span class="s s-Moment">:1h:</span> -every
-                        <span class="s s-Moment">:1m:</span> avg(<span class="s2">'value'</span>) by host
+                        <span class="s s-Moment">:1m:</span> avg(<span class="s2">'value'</span>) by host<br>
+                        <span class="o">|</span> <span class="kr">view</span> timechart
                     </div>
                     <p class="description">
-                        Return the 1 hour moving average CPU usage, updated every minute.
+                        Return the 1 hour moving average CPU usage, updated every minute and streaming live.
+                    </p>
+                </div>
+                <div class="juttle-example">
+                    <h3>
+                        Join access logs with a users database to show the top 10 site visitors by customer:
```

Comment:
Too long of a title, i saw the value of the bolded titles being generic messages on what juttle can do. Maybe "Join data from multiple sources and analyze together" or something like that? The description field already explains what the specific code snippet does.